src/views/Profile/ProfileCreation/UserName.tsx

```diff
@@ -108,11 +106,12 @@ const UserName: React.FC = () => {
   const handleConfirm = async () => {
     try {
       setIsLoading(true)
-
       const provider = ethereum as any
+      const signer = provider.getSigner()
+
       const signature = provider?.bnbSign
         ? (await provider.bnbSign(account, userName))?.signature
-        : await web3.eth.personal.sign(userName, account, null) // Last param is the password, and is null to request a signature in the wallet
+        : await signer.signMessage(userName) // Last param is the password, and is null to request a signature in the wallet
```

Comment:
Doesn't seem this comment is relevant anymore. Please remove if that is the case.